go.mod

```
@@ -685,6 +686,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/tail
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor => ./processor/transformprocessor
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/schematransformerprocessor => ./processor/schematransformerprocessor/
```

```suggestion
replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/schematransformerprocessor => ./processor/schematransformerprocessor
```

This might fix your mod download issue.

---

Oh are you serious? Let me a have a look, super silly if that is the cause.